docs/make.jl

```diff
@@ -46,22 +46,37 @@ makedocs(;
     authors="Jarrett Revels and other contributors",
     pages=[
         "Introduction" => "index.md",
-        "FAQ" => "FAQ.md",
-        "Rule configurations and calling back into AD" => "config.md",
-        "Opting out of rules" => "opting_out_of_rules.md",
-        "Writing Good Rules" => "writing_good_rules.md",
-        "Complex Numbers" => "complex.md",
-        "Deriving Array Rules" => "arrays.md",
-        "Debug Mode" => "debug_mode.md",
-        "Gradient Accumulation" => "gradient_accumulation.md",
-        "Usage in AD" => "use_in_ad_system.md",
-        "Converting ZygoteRules" => "converting_zygoterules.md",
-        "Tips for making packages work with AD" => "tips_for_packages.md",
-        "Videos" => "videos.md",
+        "The maths" => [
```

Comment:
Don't call it the `The maths` that is too intimidating, i think?

Reply:
what should we call it instead?

Reply:
Perhaps `details`?
I am not sure